server/handlers.go

```diff
@@ -517,6 +517,25 @@ func (s *LivepeerServer) setServiceURI(client eth.LivepeerEthClient, serviceURI
 	return nil
 }
 
+func (s *LivepeerServer) setMaxFaceValueHandler() http.Handler {
```

Comment:
Would be nice if you could add a unit test in `handlers_test.go`.

Reply:
Unit tests added for maxfacevalue handler.  Took some time for me to get up to speed on how to make a mock recipient in unit tests for the stub livepeer server.

Reply:
Unit tests added.